public/wp-content/themes/ColoredCow/src/scss/fonts.scss

```diff
@@ -0,0 +1,6 @@
+.about{
```

Comment:
write it like is

.font-gloria-hallelujah {
    font-family: 'Gloria Hallelujah', cursive;
}